server/index.js

```diff
@@ -31,33 +126,37 @@ app.get('/api/productImages', (req, res) => {
   });
 });
 
-app.get('*', (req, res) => {
-  res.sendFile(path.join(__dirname, '../client/dist/index.html'));
-});
-
-
 app.post('/api/productImages/create', async (req, res) => {
-  const image = req.body.image;
+  const image = req.body;
   const createImage = await db.create(image);
   res.end('done creating');
 });
 
-app.get('/api/productImages/read', async (req, res) => {
-  const id = req.query.productId;
-  const readOne = await db.read(id);
-  console.log(readOne)
-  res.send(readOne);
-});
 
-app.post('/api/productImages/update', async (req, res) => {
+app.post('/api/productImages/update',  (req, res) => {
   const id = req.query.productId;
-  const update = req.body.update;
-  const updateOne = await db.update(id, update);
+  const s3_url = req.query.s3_url;
+  console.log('what the server got', id, s3_url)
+  db.update(id, s3_url, (err, results) => {
+    if (err) {
+      console.log(err);
+      res.status(400).end();
+    } else if (!results.length) {
+      res.status(404).end();
+    } else {
+      res.status(200).send(results).end();
+    }
+  });
   res.end('done updating');
 });
 
 app.get('/api/productImages/delete', async (req, res) => {
   const id = req.query.productId;
   const deleteOne = await db.deleteOne(id);
   res.end('done deleting');
-})
\ No newline at end of file
+})
+
+// app.get('*', (req, res) => {
+//   console.log('station 2')
```

Comment:
You may want to go through your files and confirm which console.logs are relevant/irrelevant/to be deleted before deploying.